### PR TITLE
Calculator first-milestone

### PR DESCRIPTION
**Pull request by Alexoid1**
Calculator with react, Set-up complete linters add big.js and props-types.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,25 +1,41 @@
...
+    "airbnb",
+    "plugin:react/recommended"
+  ],
+  "plugins": [
+    "react"
+  ],
   "rules": {
-    "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
-    "react/react-in-jsx-scope": "off",
+    "react/jsx-filename-extension": [
```
Comment (mosaaleb) [inline .eslintrc.json]: - [x] There is another rule `"react/react-in-jsx-scope": "off"` we need to add to comply with the [linters-config](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json) provided by Microverse.

File: src/App.js [hunk 2]
```diff
@@ -1,7 +1,10 @@
+import React from 'react';
+import './App.css';
+
 function App() {
   return (
     <div className="App">
-      <h1>Hello</h1>
+      <h1 className='title'>Hello</h1>
```
Comment (mosaaleb) [inline src/App.js]: - [x] I got an eslint error here that prevents the app from compiling:

Screenshot from 2021-01-16 00-46-46
Kindly fix the linter issue.

**Review by mosaaleb — CHANGES_REQUESTED** [review]
#### Changes required

Well done @Alexoid1 :clap:. Your app setup is nearly done. However, there are multiple notes to consider before moving on. Kindly check out my comments below.

- [ ] There is an issue raised by GitHub Actions.  Make sure that all our checks (styelint and eslint) are green.

Please handle these requirements and submit a new code review.

##### Muhammad Ebeid

File: src/App.js (hunk 2 above)
Comment (Alexoid1) [reply src/App.js]: now I realize that all the issue is because i did't use double quotes

**Review by sumancrest0001 — APPROVED** [review]
### Status: Approved :+1: :100: 

**Well done @Alexoid1,
I saw that you fixed the linter configuration. Now, You can move to the next milestone.
All the best for the next milestone.**

**Happy coding !!! :slightly_smiling_face:**

**Comment by sumancrest0001** [comment]
[Required] Please follow the guideline to write commit messages. For example, Commit messages should not be the same and it should be descriptive. Commit messages should be capitalized. [Reference](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)

**Pull request merged**